Sources/HTTPParser/http_parser.swift

```diff
@@ -176,7 +312,7 @@ private let F_SKIPBODY: http_flags              = 1 << 6
 private let F_CONTENTLENGTH: http_flags         = 1 << 7
 
 /* Map for errno-related constants */
-public enum http_errno: String, Error {
+public enum http_errno: StaticString, Error {
```

Comment:
This should not be a String in the first place but just an Int num. You never use those cases as Strings anyways but the description property.